conda/recipes/custreamz/meta.yaml

```
@@ -38,19 +39,24 @@ requirements:
     - python
     - python-confluent-kafka >=1.9.0,<1.10.0a0
     - cudf_kafka ={{ version }}
+    - cuda-version ={{ cuda_version }}
   run:
     - python
     - streamz
     - cudf ={{ version }}
+    - cudf_kafka ={{ version }}
     - dask >=2023.5.1
     - dask-core >=2023.5.1
     - distributed >=2023.5.1
     - python-confluent-kafka >=1.9.0,<1.10.0a0
-    - cudf_kafka ={{ version }}
+    - {{ pin_compatible('cuda-version', max_pin='x', min_pin='x') }}
 
 test:
   requires:
+    {% if cuda_major == "11" %}
     - cudatoolkit ={{ cuda_version }}
```

Is `cuda-version` sufficient for constraining `cudatoolkit` ([ref]( https://github.com/conda-forge/cuda-version-feedstock/blob/3ce1bae701079226f982b173667edc5d0a64c01f/recipe/meta.yaml#L17 ))?

```suggestion
    - cudatoolkit
```

---

Yes, but we decided not to do this in RMM's PR so I matched that here. I am hoping to avoid unexpected changes to the CUDA 11 packaging. If we want to rely on that pinning I can make a change.

---

Aligning with RMM makes sense. Let's just open an issue to follow up after

---

https://github.com/rapidsai/cudf/issues/13607